backend/auth/auth_bearer.py

```diff
@@ -27,54 +28,15 @@ def check_scheme(self, credentials):
     async def authenticate(self, token, commons):
         if os.environ.get("AUTHENTICATE") == "false":
             return self.get_test_user()
-        elif self.verify_jwt(token):
-            return self.decode_jwt(token)
-        elif await self.verify_api_key(token, commons):
-            return await self.get_user_from_api_key(token, commons)
+        elif verify_token(token):
+            return decode_access_token(token)
+        elif await verify_api_key(token, commons):
+            return await get_user_from_api_key(token, commons)
         else:
             raise HTTPException(status_code=402, detail="Invalid token or expired token.")
```

Comment:
The returned message should maybe be more personalised since "Invalid token or expired token" is not really relevant for api key

Additionally, it would be better to return 401 instead of 402 here. (WDYT @StanGirard ? 402 is the fabulous "Payment Required")